anms-ui/public/app/components/management/agents/Agents.vue

```diff
@@ -133,9 +133,17 @@ export default {
       showManageModal: false,
       agentInfo: null,
       selectAll: false,
+      cborString: undefined
     };
   },
+  props:{
+    cbor: {
+      type:String,
+      default:undefined
+    }
+  },
   mounted() {
+    this.cborString = this.cbor;
```

Comment:
This isn't necessary. We can just bind the cbor prop directly to the agentsManageModal